packages/core/src/components/axes/zoom-bar.ts

```diff
@@ -317,6 +321,10 @@ export class ZoomBar extends Component {
 				zoomBarEventType = Events.ZoomBar.SELECTION_IN_PROGRESS;
 			} else if (event.type === "end") {
 				zoomBarEventType = Events.ZoomBar.SELECTION_END;
+				// only dispatch zoom domain change event for triggering api call when event type equals to end
+				this.services.events.dispatchEvent(Events.ZoomDomain.CHANGE, {
```

Comment:
Should/Could `SELECTION_START` and `SELECTION_IN_PROGRESS` also be considered change events?

Reply:
@zvonimirfras 
I don't think so. `ZoomDomain.CHANGE` is used to notify users that the zoom domain change is completed.
When users start to drag the zoom bar handle, there will be one `SELECTION_START` and many `SELECTION_IN_PROGRESS` events. However the zoom domain is still changing, and not the users want at this moment.
After users stop the dragging, that means users want the zoom domain, then we could/should dispatch the `CHANGE` event to notify the zoom domain change.

If users really like to know the all details of the zoom bar handle, they can just list to `ZoomBar.SELECTION_XXXX` events.